internal/buildscripts/packaging/installer/install.sh

```diff
@@ -692,6 +692,9 @@ install() {
       fi
       apt-get -y update
       install_apt_package "splunk-otel-collector" "$collector_version"
+      if [ "$distro_codename" = "noble" ]; then
+          $td_agent_version = ""
+      fi
```

Comment:
```suggestion
```
You can update the `fluentd_supported` function for `ubuntu` in https://github.com/signalfx/splunk-otel-collector/blob/main/internal/buildscripts/packaging/installer/install.sh#L1036 with something like:
```
if [ "$distro_version" = "24.04" ]; then
  return 1
fi
```
Everything else will then be automatically handled for you.